packages/containers/src/DeleteResource/actions.test.js

```diff
@@ -6,7 +6,11 @@ describe('deleteResource actions', () => {
 		it('should return an action DIALOG_BOX_DELETE_RESOURCE object', () => {
 			// Given
 			const context = {
-				router: { getCurrentLocation: jest.fn(() => ({ pathname: 'currentUrl' })) },
+				store: {
+					getState: jest.fn(() => ({
+						routing: { locationBeforeTransitions: { pathname: 'currentUrl' } },
```

Comment:
To avoid having test depending on the implem of the router from an external module, we could mock it